File: openbis/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/as/dto/pat/search/PersonalAccessTokenSessionNameSearchCriteria.js
/**
 * @author pkupczyk
 */
define([ "stjs", "as/dto/common/search/StringFieldSearchCriteria", "as/dto/common/search/SearchFieldType" ], function(stjs, StringFieldSearchCriteria, SearchFieldType) {
	var PersonalAccessTokenSessionNameSearchCriteria = function() {
		StringFieldSearchCriteria.call(this, "sessionName", SearchFieldType.ATTRIBUTE);
	};
	stjs.extend(PersonalAccessTokenSessionNameSearchCriteria, StringFieldSearchCriteria, [ StringFieldSearchCriteria ], function(constructor, prototype) {
		prototype['@type'] = 'as.dto.pat.search.PersonalAccessTokenSessionNameSearchCriteria';
		constructor.serialVersionUID = 1;
	}, {
		fieldType : {
			name : "Enum",
			arguments : [ "SearchFieldType" ]
		}
	});
	return PersonalAccessTokenSessionNameSearchCriteria;
})

File: api-openbis-java/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/pat/search/PersonalAccessTokenSessionNameSearchCriteria.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.pat.search;

import ch.ethz.sis.openbis.generic.asapi.v3.dto.common.search.SearchFieldType;
import ch.ethz.sis.openbis.generic.asapi.v3.dto.common.search.StringFieldSearchCriteria;
import ch.systemsx.cisd.base.annotation.JsonObject;

@JsonObject("as.dto.pat.search.PersonalAccessTokenSessionNameSearchCriteria")
public class PersonalAccessTokenSessionNameSearchCriteria extends StringFieldSearchCriteria
{

    private static final long serialVersionUID = 1L;

    public PersonalAccessTokenSessionNameSearchCriteria()
    {
        super("sessionName", SearchFieldType.ATTRIBUTE);
    }

}
